docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/entrypoints/global/ncids-minimal.ts

```diff
@@ -19,4 +20,7 @@ document.addEventListener('DOMContentLoaded', () => {
 
 	//Init the site alert
 	usaSiteAlertInit();
+
+	// Init InlineVideo here, unfortunately
+	inlineVideoInit();
```

Comment:
Because this ticket is just adding the video to Drupal and not really addressing analytics, we could probably remove the changes from `ncids-minimal.ts` and have this added when analytics are being done.